src/js/apps/patients/worklist/worklist_sort.js

```diff
@@ -26,12 +26,17 @@ const SortOption = Backbone.Model.extend({
     const fieldName = this.get('field_name');
     const keys = this.get('field_key').split('.');
     const direction = this.get('direction');
+    const type = this.get('type');
```

Comment:
we really shouldn't use `type` as an attribute with jsonapi.  @nmajor25

Reply:
Oh yeah. Technically it's a reserved word. We could use "kind" or "compare" instead...?

Reply:
Created a new [shortcut story](https://app.shortcut.com/roundingwell/story/32726) to update this.

Reply:
`sort_type`?